frontend/src/views/teacher/submissions/ShowSubmissionDialog.vue

```diff
@@ -0,0 +1,142 @@
+<template>
```

Comment:
would be possible to have a parametric component that is reused for the teacher and the student?